.circleci/config.yml

```diff
@@ -0,0 +1,100 @@
+version: 2
+jobs:
+  build:
+    docker:
+      # base image
+      - image: ubuntu:14.04
+      # statsd
+      - image: kamon/grafana_graphite
+      - image: atlassianlabs/localstack:0.4.0
+        environment:
+          - SERVICES=s3:5000
+          - DEFAULT_REGION=us-east-1
+          - HOSTNAME=localstack-s3
+      - image: postgres:9.4
+        environment:
+          # Create the superuser account
+          - POSTGRES_USER=postgres
+          - POSTGRES_PASSWORD=aPassword
+          - POSTGRES_DB=breakpad
+      - image: elasticsearch:1.4.5
+      - image: rabbitmq:3.6.9
+        environment:
+          - RABBITMQ_DEFAULT_USER=rabbituser
+          - RABBITMQ_DEFAULT_PASS=rabbitpwd
+          - RABBITMQ_DEFAULT_VHOST=rabbitvhost
```

Comment:
Right, this is the circley way of doing things as opposed to the docker-composey way. I haven't ripped out `make dockertest` but I think it will become just a single `docker` call.